### PR TITLE
Feature/manual update

### PR DESCRIPTION
**Pull request by DanielReid**
Be good to get some eyes on this lads

### CODE REVIEW

File: app/manual/shared.html [hunk 1]
```diff
@@ -1,35 +1,480 @@
...
+  abstract weights which result from preference elicitation. The trade off table
+  below the weights table does so by showing how much each secondary criterion
+  should change to be compensate for a certain change in a reference criterion.
+  Any criterion can be chosen as the reference, and the values between which
+  this reference criterion changes can also be set by the user.
+</p>
+<div class="figure">
+  <img src="img/tradeOffTable.png" />
+  <div class="caption">
+    The trade off table, showing how much each secondary criterion would have to
```
Comment (chinook25) [inline app/manual/shared.html:315]: Maybe use an example picture in which one of the values goes into the absurd (and thus showing the warning)? I don't think it's really necessary, but might be nice.

File: app/manual/shared.html [hunk 2]
```diff
@@ -1,35 +1,480 @@
...
+  values in the measurements table. There are many possible reasons to want to
+  do so, e.g. because you know that one study underreported certain incidences
+  and you want to see how this might affect the benefit-risk balance. After
+  adjusting one or more measurements, the 'Recalculate' button can be pressed to
+  show a second value profile plot next to the first one, allowing easy
+  comparison between the two cases and showing how the value of each alternative
+  would change if the data were different.
 </p>
+
+<!-- FIXME: update figure  -->
```
Comment (chinook25) [inline app/manual/shared.html:372]: fixme :)

File: app/manual/shared.html [hunk 3]
```diff
@@ -1,35 +1,480 @@
...
+
+<div class="figure">
+  <img src="img/smaa1.png" />
+  <div class="caption">SMAA, ranking part.</div>
+</div>
+
+<div class="figure">
+  <img src="img/smaa2.png" />
+  <div class="caption">SMAA, central weights.</div>
+</div>
```
Comment (chinook25) [inline app/manual/shared.html:431]: There should be a paragraph or sentence explaining the central weights table.

Comment (chinook25) [inline app/manual/shared.html:431]: SMAA images are outdated (still show heatmaps)

File: app/manual/shared.html [hunk 4]
```diff
@@ -1,35 +1,480 @@
...
+
+<!-- theoretical section -->
+<h2 id="mcda-benefit-risk-analysis">Benefit-risk analysis</h2>
+<p>
```
Comment (chinook25) [inline app/manual/shared.html:435]: I would put a link to this in the suggested application summary at the top. Sth. like "For a theoretical background of the application, see this section".

File: app/manual/shared.html [hunk 5]
```diff
@@ -68,439 +513,459 @@ <h3 id="mcda-running-example">Running example</h3>
...
-<p>
-  In figure 9 we see the first part of the deterministic analysis part of the benefit-risk interface. Here we can
-  adjust the effect sizes to gain further insight in what ranges of effects would tip the scales in favour of specific
-  criteria. In the example figure, we have increased the effectiveness of Regorafenib upwards, and the total value plot
-  shows that this changes the total value to now be higher than placebo for the given preferences.
+
+<h2 id="mcda-smaa-theory">Stochastic multi-criteria acceptability analyis</h2>
+<p>
+  We here cite the most important parts of our publication on SMAA. Please see
+  the original paper for full details.
```
Comment (chinook25) [inline app/manual/shared.html:933]: link?

File: app/manual/shared.html [hunk 6]
```diff
@@ -68,439 +513,459 @@ <h3 id="mcda-running-example">Running example</h3>
...
-  In figure 10 we see the second, one-way sensitivity, part of the deterministic analysis benefit-risk interface. Here
-  we can see how the total value of each criterion changes as either the effect size or effect weight changes.
+  For given (exact) values of \(x\) and \(w\) , the rank of each alternative is
+  defined as an integer from the best rank (1) to the worst rank (\(m\)) by
+  means of a ranking function rank(\(i, \xi, w)\). The main decision aiding
+  measure is the rank acceptability index , denoted by \(b^r_i\). It describes
+  the share of all possible values of the weight vector \(w\) and criteria
+  measurements \(x\) for which rank(\(i, \xi, w) = r\). For example, \(b^5_2 =
+  0.3\) means alternative 2 has fifth rank acceptability 0.3. The preferred
+  (best) alternatives are those with high acceptabilities for the best ranks
```
Comment (chinook25) [inline app/manual/shared.html:968]: Missing . at the end.

**Review by chinook25 — COMMENTED** [review]
Console throws errors;
Clicking side bar scrolls the corresponding header under the top bar (haven't we solved that before?)

File: public/lexicon.json [hunk 7]
```diff
@@ -176,7 +176,7 @@
   },
   "sensitivity-measurements": {
     "title": "Measurements sensitivity analysis",
-    "text": "How the value of a specific alternative changes as the weight of a specific criterion changes between 0 and 1. All other alternatives' values are kept constant at the current default for comparison.",
+    "text": "How the value of a specific alternative changes as the measured value of a specific criterion changes between 0 and 1. All other alternatives' values are kept constant at the current default for comparison.",
     "link": "/manual.html#mcda-deterministic-analysis"
   },
```
[the remark this answers is not in the document]
Comment (DanielReid) [reply public/lexicon.json:181]: the anchor update is coming https://trello.com/c/tl5vYKtS/3524-mcda-update-help-context-links

File: app/manual/shared.html [hunk 8]
```diff
@@ -1,35 +1,480 @@
...
+  Matching elicitation is a multi-step process that determines the trade-offs
+  between a primary criterion and all other secondary ones. The user is first
+  asked which criterion they judge as most important, after which they are
+  presented with a series of questions, one per secondary criterion. Each
+  question presents two alternative situations. The first ('alternative A') has
+  the worst possible value, given the defined scale ranges, of the primary
+  criterion, and the best possible value for the secondary criterion.
+  Alternative B is the same but has the worst possible value for the seconday
+  criterion (and therefore is by definition worse than alternative A). The user
+  is asked to adjust the value of most important criterion in alternative B so
```
Comment (kzalite) [inline app/manual/shared.html:245]: of THE most important criterion?

File: app/manual/shared.html [hunk 9]
```diff
@@ -1,35 +1,480 @@
...
+  that each scenario is approximately equivalent. Note that there is no
+  uncertainty in the elicited preferences
```
Comment (kzalite) [inline app/manual/shared.html:247]: missing trailing .

File: app/manual/shared.html [hunk 10]
```diff
@@ -1,35 +1,480 @@
...
+  </div>
+</div>
+
+<h4 id="mcda-tradeoff-table">Trade off table</h4>
+
+<p>
+  For many users, it is helpful to illustrate the concrete consequences of the
+  abstract weights which result from preference elicitation. The trade off table
+  below the weights table does so by showing how much each secondary criterion
+  should change to be compensate for a certain change in a reference criterion.
```
Comment (kzalite) [inline app/manual/shared.html:308]: 'to be compensate' skip 'be' ?

File: app/manual/shared.html [hunk 11]
```diff
@@ -1,35 +1,480 @@
...
-  treatment of type 2 diabetes mellitus (DM2). The indication for which the benefit-risk assessment is conducted is the
-  use of rimobanant as an add-on therapy to either metformin or sulphonylurea monotherapy, and the patient population
-  consists of overweight or obese DM2 patients who remained inadequately controlled after having been treated with
-  either of these established first-line therapies for a period of at least 6 months. The alternatives under
-  consideration are rimonabant 5 mg/day, rimonabant 20 mg/day, and placebo, and the criteria based on which the
-  assessment is conducted are the change in fasting plasma glucose (favourable effect) and the incidence in
-  hypoglyceamia (unfavourable effect). The performance of the three treatments on these two criteria is summarized
-  here:
+  <a href="#mcda-smaa-theory">SMAA</a> explores the possibility space (that is,
+  the N-dimensional space with each alternative's measurements and and weights
```
Comment (kzalite) [inline app/manual/shared.html:406]: double and

File: app/manual/shared.html [hunk 12]
```diff
@@ -1,35 +1,480 @@
...
+  <a href="#mcda-deterministic-analysis">deterministic</a> page to calculate the
+  value of each alternative for this sampled situation. The set of values for
+  each sampled combination, when taken as a whole, can be analysed to show in
+  what fraction of the possibility space, for example, a given alternative
+  outperforms all others (i.e. has rank 1 in the rank acceptability plot). For
+  more detail, see the
+  <a href="#mcda-smaa-theory">theoretical section on SMAA</a>.
+</p>
+
+<div class="figure">
```
Comment (kzalite) [inline app/manual/shared.html:423]: Interface has changes a bit now

**Review by kzalite — COMMENTED** [review]
Reads well

File: app/manual/shared.html (hunk 4 above)
Comment (DanielReid) [reply app/manual/shared.html:435]: line 9

File: app/manual/shared.html [hunk 13]
```diff
@@ -1,35 +1,480 @@
...
+  On the overview tab, you can edit all aspects of your workspace that are not
+  directly involved in analysis (since this would lead to inconsistencies). This
+  includes the names of alternatives, criteria and data sources, the order in
+  which they appear in tables and figures and their references, strength of
+  evidence and uncertainties where applicable. To change the order of elements
+  in a list, use the up and down arrows on their left side.
+</p>
+<h4 id="mcda-overview-criteria">Criteria, data sources, alternatives</h4>
+<p>
+  Criteria are what is used to compare the various alternatives in a
```
[the remark this answers is not in the document]
Comment (DanielReid) [reply app/manual/shared.html:31]: i think it's correct

**Review by kzalite — APPROVED** [review]

**Pull request merged**